src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,93 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private int currentCapacity;
+    private int threshold;
```

Comment:
useless variables, you can use table.length and table.length * LOAD_FACTOR
```suggestion
    private int currentCapacity;
    private int threshold;
```